site/content/docs/mdx.md

```diff
@@ -134,9 +134,7 @@ The above MDX code renders as:
 
 ---
 
-import { MyComponent } from 'components/custom/MyComponent.jsx'
-
-<MyComponent list={["cat", "dog", "parrot"]} />
+REMOVED CODE
```

Comment:
Are those REMOVED CODE intentionally left here?

Reply:
Yes, I've removed any JSX stuff from this page as it would break the build otherwise, as it is not supported in mdx-next-remote. But the page is unlined from the docs index page, so it's not discoverable. Just didn't want to remove it just yet.